runtime/etc/init/opa.conf

```diff
@@ -0,0 +1,27 @@
+description "Open Policy Agent"
+
+stop on shutdown
+
+respawn
+
+script
+      eval $(/opt/taupage/bin/parse-yaml.py /meta/taupage.yaml "config")
+      export AVAIL_ZONE="$(curl --fail -s http://169.254.169.254/latest/meta-data/placement/availability-zone)"
+      export REGION="$(echo \"$EC2_AVAIL_ZONE\" | sed -e 's:\([0-9][0-9]*\)[a-z]*\$:\\1:')"
```

Comment:
`$AVAIL_ZONE`. You also don't need to escape the quote characters, this will make your YAML invalid.

Reply:
actually we referred https://github.com/zalando-stups/taupage/blob/master/runtime/opt/taupage/init.sh#L76

Reply:
I see. That code was unfortunately not updated, but I don't think we should use legacy hard to read features (such as backticks) for new code.